CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @digitalbazaar/cborld ChangeLog
 
+## 6.1.0 - 2023-xx-xx
+
+### Added
+- Add support for compressing arbitrary URLs. Any id which is in the
+  `appContextMap` will be compressed as a single number before using other URL
```

Comment:
We should add an alias for `appContextMap` and start using it -- and deprecate the name `appContextMap` and remove it in the next version.

Maybe `appTermMap`, `appTermIdMap`, or `appStringMap` ... open to alternatives.